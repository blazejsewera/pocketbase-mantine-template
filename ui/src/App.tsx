import { MantineProvider, Text } from "@mantine/core";

export default function App() {
  return (
    <MantineProvider withGlobalStyles withNormalizeCSS>
      <Text>Welcome to Mantine!</Text>
    </MantineProvider>
  );
}
