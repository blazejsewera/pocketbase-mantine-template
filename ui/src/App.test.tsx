import App from './App'
import { snap } from './util/snapshot'

it('renders properly', () => {
  const s = snap(<App />)
  expect(s).toMatchSnapshot()
})
