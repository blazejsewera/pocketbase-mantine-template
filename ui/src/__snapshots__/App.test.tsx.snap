// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  className="App"
>
  <div>
    <a
      href="https://vitejs.dev"
      target="_blank"
    >
      <img
        alt="Vite logo"
        className="logo"
        src="/vite.svg"
      />
    </a>
    <a
      href="https://reactjs.org"
      target="_blank"
    >
      <img
        alt="React logo"
        className="logo react"
        src="test-file-stub"
      />
    </a>
  </div>
  <h1>
    Vite + React
  </h1>
  <div
    className="card"
  >
    <button
      onClick={[Function]}
    >
      count is 
      0
    </button>
    <p>
      Edit 
      <code>
        src/App.tsx
      </code>
       and save to test HMR
    </p>
  </div>
  <p
    className="read-the-docs"
  >
    Click on the Vite and React logos to learn more
  </p>
</div>
`;
